Automattic/woocommerce-payments
# Load currencies from WooCommerce Core

Fixes #2251 

#### Changes proposed in this Pull Request

* Use the currencies registered on WooCommerce Core instead of the deprecated `getCurrencyData` method from `@woocommerce/currency` package. 









#### Testing instructions


The best way I found to test it is to edit the `locale-info.php` on the WooCommerce plugin directly:

* Open `wp-contents/plugins/woocommerce/i18n/locale-info.php`.
* Find the first occurrence of the `USD` currency, to me it's the `AS` country.
* Change the `short_symbol` key to anything else, such as `$$`.

<img width="509" alt="Screen Shot 2022-03-10 at 10 22 19 PM" src="https://user-images.githubusercontent.com/1693223/157783608-7a4f29df-5ca6-469a-b7f2-0040186393dc.png">

* Open the WooCommerce Payments Overview page, the symbol you entered should be showing there.

<img width="699" alt="Screen Shot 2022-03-10 at 7 01 56 PM" src="https://user-images.githubusercontent.com/1693223/157762507-4f173ac4-d4b3-4a28-bf80-a54a85205898.png">

It's worth noting that the EUR symbol € on the WooCommerce Core file has the `currency_pos` property set to `right_space`, so all EUR currencies on WooCommerce Payments will change from `€ 100,00` to `100,00 €`.



-------------------


**Post merge**
